File: Resolvendo Algoritmos/Idade em Dias/idade-em-dias.cs
using System;

namespace Desafios.ResolvendoAlgoritmos {
    class IdadeEmDias {
        static void Main(string[] args) {
            var dias = int.Parse(Console.ReadLine());
            var anos = dias / 365;
            dias = dias % 365;
            var meses = dias / 30;
            dias = dias % 30;

            Console.WriteLine($"{anos} ano(s)");
            Console.WriteLine($"{meses} mes(es)");
            Console.WriteLine($"{dias} dia(s)");
        }
    }
}

File: Solução de problemas básicos em Java/Idade em Dias/idade-em-dias.java
import java.util.Scanner;

public class IdadeEmDias {
    
    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        int idadeDias = input.nextInt();
        int anos = idadeDias / 365;
        int meses = ((idadeDias % 365) / 30);
        int dias = ((idadeDias % 365) % 30);

        System.out.println(anos + " ano(s)");
        System.out.println(meses + " mes(es)");
        System.out.println(dias + " dia(s)");

        input.close();
    }
 
} 